src/tokens/ERC20RebaseDistributor.sol

```diff
@@ -98,8 +112,10 @@ abstract contract ERC20RebaseDistributor is ERC20 {
         uint256 balance = ERC20.balanceOf(account);
         uint256 shares = (balance * START_REBASING_SHARE_PRICE) /
```

Comment:
would be nice if this logic was in a helper function

Reply:
In ERC4626 there are these 2 view functions : 
```
function convertToShares(uint256 assets) public view returns (uint256 shares)
function convertToAssets(uint256 shares) public view returns (uint256 assets)
```
I think that's close to what you're saying, but I don't want to implement it because I'm saving SLOAD by reusing a cached `rebasingSharePrice` in 2 out of the 3 places where this formula is used

Reply:
Could just be an `internal` function that takes the share price as an argument to allow cached values to be re-used.

Reply:
I'm adding `_balance2shares` and `_shares2balance` internal helpers